tests/adapter/dbt/tests/adapter/simple_seed/test_seed.py

```diff
@@ -229,32 +234,24 @@ class TestSimpleSeedWithBOM(SeedConfigBase):
     @pytest.fixture(scope="class", autouse=True)
     def setUp(self, project):
         """Create table for ensuring seeds and models used in tests build correctly"""
-        project.run_sql_file(project.test_data_dir / Path("seed_expected.sql"))
+        project.run_sql(seeds__expected_sql)
 
-        # manual copy because seed has a special and tricky-to-include unicode character at 0
-        shutil.copyfile(
-            project.test_data_dir / Path("seed_bom.csv"),
-            project.project_root / Path("seeds") / Path("seed_bom.csv"),
-        )
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {"seed_bom.csv": seed__bom_csv}
 
     def test_simple_seed(self, project):
-        results = run_dbt(["seed"])
-        assert len(results) == 1
-
-        # encoding param must be specified in open, so long as Python reads files with a
-        # default file encoding for character sets beyond extended ASCII.
-        with open(
-            project.project_root / Path("seeds") / Path("seed_bom.csv"), encoding="utf-8"
-        ) as fp:
-            assert fp.read(1) == BOM_UTF8.decode("utf-8")
```

Comment:
Couldn't exactly map this to the new test. CC: @VersusFacit 
```
codec = "utf-8"
encoded = seed__bom_csv.encode(codec)
assert seed__bom_csv == encoded.decode(codec)
```
is what can possibly replace but it's not great.